app/controllers/concerns/course_basic_detail_concern.rb

```diff
@@ -4,6 +4,7 @@ module CourseBasicDetailConcern
   included do
     decorates_assigned :course
     before_action :build_provider, :build_new_course, only: %i[new continue]
+    before_action :get_previous_course_creation_params, only: %i[new continue]
```

Comment:
We have a `build` vs `get` approach in our code base so it's worth being consistent.